lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -43,21 +43,21 @@
     }
 
   .icon {
-    @include font-size(14);
+    font-size: 14px;
     @include margin-right($baseline/4);
   }
 }
 
 .forum-nav-browse-current {
-  @include font-size(12);
+  font-size: 12px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 12px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.